docs/toc.md

```diff
@@ -390,6 +395,8 @@
 # [Visual Basic Guide](visual-basic/index.md)
 ## [Get Started](visual-basic/getting-started/index.md)
 ### [What's New for Visual Basic](visual-basic/getting-started/whats-new.md)
+### [Build a Visual Basic Hello World app with .NET Core in Visual Studio 2017](core/tutorials/w)
+### [Build a class library with Visual Basic and .NET Core in Visual Studio 2017](core/tutorials/vb-library-with-visual-studio.md)
```

Comment:
why do we have the same link in two places? For example, you'll end up with two locations on the TOC highlighted and screen will always sync on the first one, even if you click inside the VB toc. e.g. https://review.docs.microsoft.com/en-us/dotnet/core/tutorials/vb-library-with-visual-studio?branch=pr-en-us-2916